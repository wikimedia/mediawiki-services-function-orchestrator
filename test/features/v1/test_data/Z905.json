{
	"Z1K1": {
		"Z1K1": "Z9",
		"Z9K1": "Z7"
	},
	"Z7K1": {
		"Z1K1": {
			"Z1K1": "Z9",
			"Z9K1": "Z8"
		},
		"Z8K1": [
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z1"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z805K1"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": []
				}
			}
		],
		"Z8K2": {
			"Z1K1": "Z9",
			"Z9K1": "Z1"
		},
		"Z8K3": [],
		"Z8K4": [
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z14"
				},
				"Z14K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z805"
				},
				"Z14K4": {
					"Z1K1": "Z6",
					"Z6K1": "Z905"
				}
			}
		],
		"Z8K5": {
			"Z1K1": "Z9",
			"Z9K1": "Z805"
		}
	},
	"Z805K1": {
		"Z1K1": {
			"Z1K1": "Z9",
			"Z9K1": "Z60"
		},
		"Z60K1": {
			"Z1K1": "Z6",
			"Z6K1": "tlh"
		}
	}
}
