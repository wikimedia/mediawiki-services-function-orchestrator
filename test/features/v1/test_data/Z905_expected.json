[
	{
		"Z1K1": {
			"Z1K1": "Z7",
			"Z7K1": "Z882",
			"Z882K1": "Z39",
			"Z882K2": "Z1"
		},
		"K1": {
			"Z1K1": "Z39",
			"Z39K1": "Z1K1"
		},
		"K2": [
			{
				"Z1K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z882",
					"Z882K1": "Z39",
					"Z882K2": "Z1"
				},
				"K1": {
					"Z1K1": "Z39",
					"Z39K1": "Z1K1"
				},
				"K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z9"
				}
			},
			{
				"Z1K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z882",
					"Z882K1": "Z39",
					"Z882K2": "Z1"
				},
				"K1": {
					"Z1K1": "Z39",
					"Z39K1": "Z9K1"
				},
				"K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z60"
				}
			}
		]
	},
	{
		"Z1K1": {
			"Z1K1": "Z7",
			"Z7K1": "Z882",
			"Z882K1": "Z39",
			"Z882K2": "Z1"
		},
		"K1": {
			"Z1K1": "Z39",
			"Z39K1": "Z60K1"
		},
		"K2": [
			{
				"Z1K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z882",
					"Z882K1": "Z39",
					"Z882K2": "Z1"
				},
				"K1": {
					"Z1K1": "Z39",
					"Z39K1": "Z1K1"
				},
				"K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z6"
				}
			},
			{
				"Z1K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z882",
					"Z882K1": "Z39",
					"Z882K2": "Z1"
				},
				"K1": {
					"Z1K1": "Z39",
					"Z39K1": "Z6K1"
				},
				"K2": "tlh"
			}
		]
	}
]
